monkey/infection_monkey/exploit/weblogic.py

```diff
@@ -20,8 +20,8 @@
 LOG = logging.getLogger(__name__)
 # How long server waits for get request in seconds
 SERVER_TIMEOUT = 4
-# How long should be wait after each request in seconds
-REQUEST_DELAY = 0.0001
+# How long should we wait after each request in seconds
+REQUEST_DELAY = 0.1
```

Comment:
Any good reason to change this? And specifically in this branch?

Reply:
No. Needed this bugfix to test and forgot to revert. Bugfix will be included in new weblogic exploiter.